timify-api/route/users.js

```diff
@@ -0,0 +1,66 @@
+import express from 'express';
+import bcrypt from 'bcrypt';
+import { User } from '../models/user.js';
+import { Op } from 'sequelize';
+
+const router = express.Router();
+
+// Route for user registration
+router.post('/users', async (req, res) => {
+  const { first_name, last_name, username, password, email } = req.body;
+
+  try {
+
+    const existingUser = await User.findOne({
+      where: {
+        [Op.or]: [{ username }, { email }]
+      }
+    });
+    console.log("user not found", existingUser)
+
+    if (existingUser) {
+      return res.status(400).json({ error: 'Username or email already exists' });
+    }
+
+    const hashedPassword = await bcrypt.hash(password, 10);
```

Comment:
same question as elsewhere about 10, if it's used in two different places it should be defined as one variable that is used everywhere instead.